reference/best_practices/index.rst

```diff
@@ -0,0 +1,15 @@
+Best Practices
+==============
+
+Here is our recommendations as developers and integrators.
+It results from our experience in the realisation of Akeneo PIM projects.
+
+The coding style standards is another interesting part you can read too: :doc:`/contributing/standards`.
```

Comment:
If you wish to know more about coding style standards, you can have a lookg at:

I think you should put this sentence in the very end of the summary as it doesn't have any special interest having it at the very beginning.